src/components/Link/index.ts

```diff
@@ -0,0 +1 @@
+export { LinkTemplate } from "./Link";
```

Comment:
```suggestion
export { Link } from "./Link";
```

Reply:
I use Link from react-router-dom, so if i change "LinkTemplate" to "Link" i will have mistake. 

I already tried to remove this word "template". There are only two ways:
1. keep word "template"
2. change Link(from react router dom) to usual <a>

Reply:
you can react-router link import with another name, this we already dicussed on your previous project. https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/import